### ISSUE TITLE
Images are a year out of date

### ISSUE THREAD

**Issue by aaronmgriffin** [bubuntux/nordlynx#172]
### **Describe the issue**
Images on both dockerhub and github packages are very outdated, and the repo is behind the parent wireguard repo. Please update! 

### **Expected behavior**
The readme suggests weekly/monthly image updates on :edge and :latest tags

**Comment by KaKi87** [comment]
Bump

**Comment by jaycuse** [comment]
Created a PR to update base image. Hopefully this can get us a new build.
https://github.com/bubuntux/nordlynx/pull/178

**Comment by jaycuse** [comment]
Unfortunately the build is still failing.
```
Error: buildx failed with: ERROR: failed to solve: ghcr.io/linuxserver/baseimage-alpine:3.20: failed to resolve source metadata for ghcr.io/linuxserver/baseimage-alpine:3.20: no match for platform in manifest: not found
```

Not sure why it's not finding the image because I see `3.20` as listed there: https://github.com/linuxserver/docker-baseimage-alpine/pkgs/container/baseimage-alpine

**Comment by KaKi87** [comment]
Hi,
Thank you !
How do I make requests through it ?

**Comment by KaKi87** [comment]
@arkandias 🙏

**Comment by KaKi87** [comment]
Bump

**Comment by KaKi87** [comment]
Bump

**Comment by godfuture** [comment]
stale bot is really aggressive...

**Comment by KaKi87** [comment]
Yep, even  though there are several constructive cases against its use :

- January 2021 : https://blog.benwinding.com/github-stale-bots/
- October 2021 : https://drewdevault.com/2021/10/26/stalebot.html
- February 2023 : https://fvsch.com/stale-bots

**Comment by godfuture** [comment]
In the end they have to fight duplicate tickets, but only if this repo is really meant as serious place to improve. Otherwise this is all show.

**Comment by KaKi87** [comment]
Bump @bubuntux

**Comment by KaKi87** [comment]
@bubuntux 😭

**Comment by bubuntux** [comment]
happy to accept merge requests

**Comment by bubuntux** [comment]
Repo archieved; closing as stale
